scripts/one-off/webhook-settings-migration/migrate.ts

```diff
@@ -0,0 +1,48 @@
+import './loadEnv.js';
```

Comment:
better doing that in a DB migration (without transaction), otherwise self-hosted and entreprise won't be migrated

Reply:
Indeed, good call. Updated with https://github.com/NangoHQ/nango/pull/2302/commits/139ecad30edb7e906df644551557aef12ae448f2

Reply:
do you want to delete this script?